semgrep/semgrep/core_runner.py

```diff
@@ -246,16 +244,14 @@ def _run_core_command(
                 str(self._timeout),
                 "-max_memory",
                 str(self._max_memory),
+                "-json_time",
             ]
 
             equivalences = rule.equivalences
             if equivalences:
                 self._write_equivalences_file(equiv_file, equivalences)
                 cmd += ["-equivalences", equiv_file.name]
 
-            if self._report_time:
```

Comment:
Always asks semgrep-core to report timing information even if we don't print them out (user did not specify `--time`)

Reply:
This makes the _report_time option unnecessary in core_runner.py